### ISSUE TITLE
Notes on what we did for Rogue Scholar integration

### ISSUE THREAD

**Issue by maelle** [ropensci/roweb3#663]
(For a possible tech note)

With a lot of help from @mfenner,

- Reformated our JSON index, in particular to add info about authors and about relationships (cross-posts, translations).
- Submitted said index to Rogue Scholar
- Added code to our layouts to find the DOI from Rogue Scholar API if there's one. If there's one it's included in the post source's head and in the sidebar with a copy-paste button. If there's not one yet, in the sidebar we use a fallback URL.

**Comment by noamross** [comment]
I was wondering about this. So this means that the first time site is built, there's no DOI, but in subsequent builds it is able to get the DOI?

**Comment by maelle** [comment]
Yes, and we build the website once a day anyway (to make sure the events page is updates for instance) https://github.com/ropensci/roweb3/blob/main/.github/workflows/main.yml

**Comment by maelle** [comment]
we're switching to a different approach

- store DOI in YAML (DOI from Rogue Scholar from old posts, generate one for new posts before publishing)
- use that DOI in the HTML and JSON templates.

**Comment by maelle** [comment]
we've now added authors' ORCID to the feed when we have it. This will help with name formatting (what's the last name, what's the first name) in particular.

**Comment by mfenner** [comment]
Thanks for the update! I will need to update the Rogue Scholar API, should be done by Monday. Will check that names are formatted correctly after updating.
